docs/source/getting_started/dataset_creation/dataset_creation_gcloud_bucket.rst

```diff
@@ -1,6 +1,164 @@
 .. _dataset-creation-gcloud-bucket:
 
-Create dataset from google cloud bucket
----------------------------------------
+Create a dataset from Google Cloud Storage
+------------------------------------------
 
-TODO: Add the tutorial how to create a Lightly dataset from a gcloud bucket here.
+Lightly allows you to configure a remote datasource like
+`Google Cloud Storage <https://cloud.google.com/storage>`_ .
+In this guide, we will show you how to setup your Google Cloud Storage,
+configure your dataset to use said bucket, and only upload metadata to Lightly
+while keeping your data private.
+
+
+Setting up Google Cloud Storage
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Lightly needs to be able to create so-called
+`presigned URLs/read URLs <https://cloud.google.com/storage/docs/access-control/signed-urls>`_
+for displaying your data in your browser.
+Thus it needs at minimum to be able to read and list permissions on your bucket.
+If you want Lightly to create optimal thumbnails for you
+while uploading the metadata of your images, write permissions are also needed.
+
+Let us assume the bucket is called `lightly-datalake`.
+And let us assume the folder you want to use with Lightly is located at `projects/wild-animals/`
+
+**Setting Up a Service Account in IAM**
+
+1. Write down your project ID.
+You find it in the `gcloud console <console.cloud.google.com/home/dashboard>`_ under `Project Info`.
+
+2. Navigate to your bucket in the `google cloud storage browser <https://console.cloud.google.com/storage/browser>`_
+and from there to `projects/wild-animals/`. Copy the path, in this case
+`lightly-datalake/projects/wild-animals`.
+
+.. figure:: ./images_gcloud_bucket/screenshot_gcloud_bucket_project.jpg
+    :align: center
+    :alt: Browsing a google cloud storage bucket.
+    :width: 60%
+
+3. Navigate to the tab `Permissions`.
+Make sure that your access control is uniform.
+If it is not, change it to uniform.
+
+.. figure:: ./images_gcloud_bucket/screenshot_gcloud_uniform_access.jpg
+    :align: center
+    :alt: Ensuring a google cloud bucket has uniform access.
+    :width: 60%
+
+4. Navigate to `IAM & Admin -> Roles <https://console.cloud.google.com/iam-admin/roles>`_.
+
+- There create a new role, with the title and ID `STORAGE_READ_WRITE_LIST`.
```

Comment:
maybe name this something lightly related. LIGHTLY_DATSET_ACCESS

Reply:
I think the comma should be like this: "There, create a new role with the title ..."

Reply:
The role itself is not at all Lightly related, it is a role allowing read, write, list for any bucket. Alternatively, one could use the role Storage Admin, that would work as well.

As the role is neither about a specific bucket, nor about a dataset in this bucket, I would leave it like this.

Reply:
@philippmwirth I changed the syntax.